### ISSUE TITLE
Fatal exception of type "TypeError" when |openreferences=("true", "yes", or "missing") when used with |imagecontiner=

### ISSUE THREAD

**Issue by CrystalClearGal** [Universal-Omega/DynamicPageList3#163]
**DynamicPageList3 version**: 3.4.5
**MediaWiki version**: 1.37.2 (Miraheze)
**PHP version**: 7.4.28

TLDR: Problem observed here:
https://dpl3.miraheze.org/wiki/Display_images_used_(example)

**List of steps to reproduce** (step by step, including full links if applicable):
* Create an "Apple" page with at least one image used.
* Place **either** of following on any other page:
 ```
{{#dpl:
|imagecontainer = Apple¦Orange
|openreferences=true
}}
```
**OR**
```
{{#dpl:
|imagecontainer = Apple¦Orange
|openreferences=yes
}}
```
* Preview or save to view fatal error

### **Some notes:**
* `imagecontainer` documentation states that the required value is <code>|openreferences=true</code>.
* `linksfrom`documentation states to use <code>|openreferences=yes</code>.
* `openreferences` documentation states the correct syntax is <code>|openreferences=yes</code>.
* Neither works

**What happens?**:
Fatal exception of type "TypeError" 

**What should have happened instead?**:
Result should display.

**Browser information, screenshots and other applicable information**:
Not related, happens irregardless of browser

**Comment by Universal-Omega** [comment]
Can you please create a Phabricator task on Miraheze to simply request the stack trace for the error. If you do, please include the exception id as well so we csn get it, so that I can know the full error for the purposes of this issue.

**Comment by CrystalClearGal** [comment]
Yes, will do.

**Comment by Universal-Omega** [comment]
> Yes, will do.

Thank you!

**Comment by CrystalClearGal** [comment]
No prob, and done. :)
https://phabricator.miraheze.org/T9056

Ps. I hope it's okay that I subscribed you to the phab task, I don't really understand phab etiquette.

**Comment by Universal-Omega** [comment]
> No prob, and done. :) https://phabricator.miraheze.org/T9056
> 
> Ps. I hope it's okay that I subscribed you to the phab task, I don't really understand phab etiquette.

Thanks again! And no problem for subscribing me, but I would've automatically been subscribed anyway, as I'm automatically subscribed to almost every Phabricator task on Miraheze.

**Comment by CrystalClearGal** [comment]
Is it possible that when the 1.38 upgrade was done, the view didn't get created again? I ask because I went to mark this as resolved but re-tested just now the same thing again, or nearly the same, and am receiving the same error again (though the cause may be different). I was about to request another stack trace for it and realized, it could be the same issue on MH. If the view does exist there though, I'll happily request again the strack trace.

**Comment by Universal-Omega** [comment]
Should be fixed by https://github.com/Universal-Omega/DynamicPageList3/commit/6ca7c458be822f32905b814f1b9e9ff2dd70e035
